tests/checkbox/read-checkbox-grouping.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Group role, name and boundaries are announced when navigating through checkbox group.</title>
+<link rel="author" title="Valerie Young" href="mailto:valerie@bocoup.com">
+<link rel="help" href="https://www.w3.org/TR/wai-aria-practices-1.1/#checkbox">
+<link rel="help" href="https://w3c.github.io/aria/#checkbox">
+<link rel="help" href="https://w3c.github.io/aria/#aria-checked">
+<link rel="help" href="https://w3c.github.io/aria/#group">
+<script src="/resources/ariaatharness.js">
+<script>
+
+  openTestPage("reference/checkbox-group.html");
+
+  presentATTest({
+    mode: ["reading", "interaction"],
+    action: "navigation",
+    assertion: [
+      "The role 'group' is announced",
+      "The group's name 'Sandwich Condiments' is announced",
+      "The boundaries of the group (before the first checkbox and after the last checkbox) are communicated"
+    ],
```

Comment:
I like this approach since it is simple and intuitive, and some good suggestions were provided on the call earlier. Some of my thoughts:

1. This could end up being a rather large list of commands. To save time, it might be worthwhile to select 2 or 3 common commands and only test those for the early phases of this project. I think that would have the highest return on investment.
2. When a command fails, how will we track that specific failure (see next comment).
3. Abstracting to other assistive technologies (beyond screen readers): I took the approach of keeping the assertions generic and applying them to many technologies. For example, both screen readers and voice control software make use of a control's name, but voice control software will not convey boundaries. With this approach, some assertions might not be applicable to all AT, but it does let us display support for a specific aspect of the feature across many AT.
4. if one of the end goals is to display support data for different aria attributes, we should consider how assertions for specific attributes will be reused across tests. I'd suggest that the assertions should be defined in a centralized place, and tests should link to them as foreign keys.

Reply:
Good point about assertions. I was thinking the assertions should be more structured, at least. We might want to decide how to abstract them/write them/link to them after we write some more tests. Can you link some examples of your more generic assertions?

Reply:
Sure. Using a11yspport.io and HTML links as an example:

- [an HTML link feature that references many tests](https://a11ysupport.io/tech/html/a(href)_element)
- [The assertions are defined on the feature object (JSON)](https://github.com/accessibilitysupported/a11ysupport.io/blob/master/data/tech/html/a(href)_element.json)
- [And referenced by the tests (JSON)](https://github.com/accessibilitysupported/a11ysupport.io/blob/master/data/tests/tech/html/links/example1.json)
- Note that abstract versions of common assertions such as 'convey name', and 'convey role' [are defined further left in the build process](https://github.com/accessibilitysupported/a11ysupport.io/blob/master/src/feature-helper.js#L94). This helps with consistency and reduces redundancies. 
- My approach is slightly different than the approach we are taking here, but I think we can apply similar concepts. We will be testing the same ARIA features across many tests.